test/test_opet_coin.js

```diff
@@ -0,0 +1,145 @@
+        } catch (err) {
+            assert.equal(err, 'Error: VM Exception while processing transaction: revert', "Wrong error after try to use the airdrop with different arrays length");
+        }
+        await token.sendAirdrops(airdropAccounts, airdropAmounts);
+        let afterAirdropBalance0 = BigNumber(await token.balanceOf(airdropAccounts[0]));
+        let afterAirdropBalance1 = BigNumber(await token.balanceOf(airdropAccounts[1]));
+        assert.equal(afterAirdropBalance0.toNumber(), airdropAmounts[0], 'Wrong address balance after token airdrop for address 0');
+        assert.equal(afterAirdropBalance1.toNumber(), airdropAmounts[1], 'Wrong address balance after token airdrop for address 1');
+    });
+
+    it("test token set minter for airdrop", async function () {
+        let token = await Token.deployed();
+        let airdropAccounts = [accounts[3]];
+        let airdropAmounts = [1];
+        await token.setMinter(airdropAccounts[0], true);
+        await token.sendAirdrops(airdropAccounts, airdropAmounts, {'from': airdropAccounts[0]});
+        let afterAirdropBalance = BigNumber(await token.balanceOf(airdropAccounts[0]));
+        assert.equal(afterAirdropBalance.toNumber(), 1, 'Wrong address balance after token mint');
+    });
+});
+
+contract("Opet Token", async function(accounts) {
+    it("test send max airdrop", async function () {
+        let token = await Token.deployed();
+        let airdropAccount = [accounts[1]];
+        let airdropAmount = [BigNumber(await token.AIRDROP_SUPPLY()).toString()];
+
+        assert.equal('586365000000000', airdropAmount[0], "Wrong amount of Airdrop supply");
+        await token.sendAirdrops(airdropAccount, airdropAmount);
+        let afterAirdropBalance = BigNumber(await token.balanceOf(airdropAccount[0])).toString();
+        assert.equal(afterAirdropBalance, airdropAmount[0], 'Wrong address balance after token airdrop');
+
+        try {
+            await token.sendAirdrops(airdropAccount, [1]);
+            assert.ifError('Error, too much tokens successfuly airdropped');
+        } catch (err) {
+            assert.equal(err, 'Error: VM Exception while processing transaction: revert', "Wrong error too big amount of airdrop");
+        }
+    });
+});
+
+
+contract("Opet Token", async function (accounts) {
+    it("test transfer unable to execute", async function () {
+        let token = await Token.deployed();
+        let mintDestination = accounts[1];
+        let transferDestination = accounts[2];
+
+
+        await token.mint(mintDestination, 2);
+        try {
+            await token.transfer(transferDestination, 1, {'from': mintDestination});
+            assert.ifError('Error, transfer should be unable');
+        } catch (err) {
+            assert.equal(err, 'Error: VM Exception while processing transaction: revert', "Transfer works before token unpaused");
+        }
+
+        await token.approve(transferDestination, 1, {'from': mintDestination});
```

Comment:
Please add a comment above the block - `Test token "approve" / "transferFrom" flow`